volley/volley.py

```diff
@@ -0,0 +1,168 @@
+# Implementation of Volley
+
+import math
+import os
+import requests
+import sqlite3
+
+import sys
+sys.path.insert(0, os.path.normpath('../cache'))
+
+import ip_location_cache
+
+# Config
+LOG_DATABASE = os.path.normpath('../aggregator/aggregated_logs.db')
+
+class Volley:
```

Comment:
Hm. I'll think about that a little bit, but as for now, there should be a file that executes volley, retrieves recommendations, and makes the appropriate migrations.